modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -346,9 +362,14 @@ components:
                         additionalProperties:
                             type: string
                         type: object
+                    # @review
+                    description:
+                        Block of actions allowed by the user making the request.
```

Comment:
and here we should link to, hopefully, the actions documentation page explaining what "actions" are